Source/bmalloc/libpas/src/libpas/pas_root.c

```diff
@@ -145,6 +146,8 @@ void pas_root_construct(pas_root* root)
 
     root->baseline_allocator_table = &pas_baseline_allocator_table;
     root->num_baseline_allocators = PAS_NUM_BASELINE_ALLOCATORS;
+
+    root->root_pas_crash_report_version = get_pas_crash_report_version_ptr();
```

Comment:
Why not setting this version to `root` directly and read the field in pas_report_crash.c?